core/primitives/src/version.rs

```diff
@@ -94,21 +94,23 @@ pub enum ProtocolFeature {
     AltBn128,
     #[cfg(feature = "protocol_feature_access_key_nonce_range")]
     AccessKeyNonceRange,
+    DeleteActionRestriction,
 }
 
 /// Current latest stable version of the protocol.
 #[cfg(not(feature = "nightly_protocol"))]
-pub const PROTOCOL_VERSION: ProtocolVersion = 42;
+pub const PROTOCOL_VERSION: ProtocolVersion = 43;
 
 /// Current latest nightly version of the protocol.
 #[cfg(feature = "nightly_protocol")]
 pub const PROTOCOL_VERSION: ProtocolVersion = 106;
 
 lazy_static! {
-    static ref STABLE_PROTOCOL_FEATURES_TO_VERSION_MAPPING: HashMap<ProtocolFeature, ProtocolVersion> =
+    pub static ref STABLE_PROTOCOL_FEATURES_TO_VERSION_MAPPING: HashMap<ProtocolFeature, ProtocolVersion> =
```

Comment:
This is not needed. nightly merges stable features in

Reply:
Good point.